network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/cause_statement.html

```diff
@@ -1,10 +0,0 @@
-<div class="row mb-4">
-    <div class="col">
-        <div class="section-cause-statement tw-dark p-5 text-center">
```

Comment:
Let's delete `.section-cause-statement` related SCSS code as well.

Reply:
Addressing design teams feedback, I did not include the removal of the `cause statement` section in this PR.